src/eventHandlers/AccountMetadataEmittedEvent/handlers/handleEcosystemMainAccountMetadata.ts

```diff
@@ -207,23 +215,24 @@ async function createNewSplitReceivers({
   transaction: Transaction;
   emitterAccountId: NftDriverId;
   splitReceivers: (
-    | z.infer<typeof repoDriverSplitReceiverSchema>
+    | z.infer<typeof repoSubAccountDriverSplitReceiverSchema>
     | z.infer<typeof subListSplitReceiverSchema>
   )[];
 }) {
   const receiverPromises = splitReceivers.map(async (receiver) => {
     switch (receiver.type) {
-      case 'repoDriver':
-        assertIsRepoDriverId(receiver.accountId);
+      case 'repoSubAccountDriver':
+        // eslint-disable-next-line no-case-declarations
```

Comment:
I think you can also just surround the case block with `{ ... }` to get around this